dbt/adapters/postgres/impl.py

```diff
@@ -158,7 +158,10 @@ def _link_cached_relations(self, manifest, schemas):
                                               identifier=refed_name)
             dependent = self.Relation.create(schema=dep_schema,
                                              identifier=dep_name)
-            self.cache.add_link(dependent, referenced)
+
+            # don't record in cache if this relation isn't in a relevant schema
+            if refed_schema in schemas:
```

Comment:
@beckjake do you think we need to do a case-insensitive check here?

Reply:
Yeah, it needs to be `refed_schema.lower() in schemas`.